pom.xml

```diff
@@ -311,9 +326,9 @@
           <extensions>true</extensions>
           <configuration>
             <instructions>
-              <_exportcontents>co.cask.hydrator.plugin.batch.*;org.elasticsearch.hadoop.mr.*;
+              <_exportcontents>io.cdap.plugin.batch.*;org.elasticsearch.hadoop.mr.*;
                 org.apache.commons.lang;org.apache.commons.*;org.apache.commons.logging.*;
```

Comment:
this is a long list of common exports. Are you able to run this plugin? Are these all required to run?